src/fidesops/schemas/user.py

```diff
@@ -48,3 +49,10 @@ class UserCreateResponse(BaseSchema):
     """Response after creating a FidesopsUser"""
 
     id: str
+
+
+class PrivacyRequestReviewerResponse(BaseSchema):
```

Comment:
Nitpick but I'm not sure if including Response in this name makes sense. This would indicate that this is the schema for a response object but we're using it as the type for a field in a response object. Perhaps `PrivacyRequestReviewer` makes more sense?